src/components/Entry/Entry.js

```diff
@@ -1,9 +1,11 @@
-import React, {useState} from 'react'
-import {useNavigate} from 'react-router-dom'
+import React, { useContext, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
 import { Rating } from 'react-simple-star-rating'
 import'./Entry.css'
+import UserContext from '../UserContext/UserContext'
 
-function Entry ({setCurrentUser, id, title, artists, date, notes, image, rating, masterId}) {
+function Entry ({id, title, artists, date, notes, image, rating, masterId}) {
+  const [setCurrentUser] = useContext(UserContext);
```

Comment:
Not sure what's happening but on my end I'm getting an error whenever I add to or remove from the journal.
'Uncaught TypeError: setCurrentUser is not a function'
Maybe this is where we have to put the setter into a function?  Or maybe use effect

Reply:
Think I figured it out, values should be an object not an array.
`{setCurrentUser} = useContext(UserContext)`

Reply:
Thanks for trying things out. But that solution does not seem to to work on my end, I have to pass it in like this
`const [currentUser, setCurrentUser] = useContext(UserContext)`. Even if I don't use `currentUser`